src/features/Authentication/organisms/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from "yup";
+import { TextField } from 'formik-mui';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import CircularProgress from '@mui/material/CircularProgress';
+import { CREATE_USER } from 'mutations/CreateUser/createUser';
+import { useMutation } from 'urql';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+
+const SIGNUP_SCHEMA = Yup.object().shape({
+  firstName: Yup.string()
+    .min(2, 'Too Short!')
+    .max(70, 'Too Long!')
+    .required('Required'),
+  lastName: Yup.string()
+    .min(2, 'Too Short!')
+    .max(70, 'Too Long!')
+    .required('Required'),
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  password: Yup.string()
+    .required("Please enter your password")
+    .matches(
+      /^.*(?=.{10,})((?=.*[!@#$%^&*()\-_=+{};:,<.>]){1})(?=.*\d)((?=.*[a-z]){1})((?=.*[A-Z]){1}).*$/,
+      "Password must contain at least 10 characters, one uppercase, one number and one special case character"
+    ),
+});
+
+const INITIAL_VALUES = {
+  firstName: '',
+  lastName: '',
+  email: '',
+  password: '',
+  passwordConfirmation: '',
+}
+
+const SignUpForm = () => {
+  const [{ fetching, error }, createUser] = useMutation(CREATE_USER);
+
+  const onSubmit = async (values, { setSubmitting }, {error}) => {
+    await createUser(values).then(result => {
+      if (result.error) {
+        console.error('Oh no!', result.error.message);
```

Comment:
Is not very useful approach for error handling

Reply:
How it can be changed?

Reply:
You need to parse error message and display it near the form